examples/gno.land/p/demo/blog/util.gno

```diff
@@ -5,3 +5,26 @@ import "strings"
 func breadcrumb(parts []string) string {
 	return "# " + strings.Join(parts, " / ") + "\n\n"
 }
+
+func (p postSlice) GetPost(slug string) *Post {
```

Comment:
maybe change this to `BySlug`

Reply:
Changed it to GetPostBySlug for clarity.